src/utilities/util.ts

```diff
@@ -134,3 +135,7 @@ export function hasMatchWithIndex<T>(list1: T[], list2: T[]): [number, number] {
 
     return matchingIndices;
 }
+
+export function isImportable(object: unknown): object is Importable {
```

Comment:
This is TypeScript's way of checking if something is an instance of some interface. You literally have to manually check the properties. At least the return type is a compiler recognized type so you only have to perform the check once within a block of code.